File: openapi/code_samples/csharp/api_company_bankrupt_activeYears.cs
using Microsoft.AspNetCore.Mvc;
using Swashbuckle.AspNetCore.Annotations;

[ApiController]
[Route("api/[controller]")]
public class CompanyController : ControllerBase
{
    [HttpGet]
    [Route("bankrupt-companies-active-years")]
    [SwaggerOperation(Summary = "Retrieves the active years of bankrupt companies.")]
    [ProducesResponseType(typeof(PageBankruptCompaniesActiveYearsView), 200)]
    [ProducesResponseType(typeof(Fault), 400)]
    public IActionResult GetBankruptCompaniesActiveYears(
        [FromQuery(Name = "activityYears")] int activityYears = 3,
        [FromQuery(Name = "caen")] string caen = "",
        [FromQuery(Name = "cui")] string cui = "",
        [FromQuery(Name = "county")] string county = "",
        [FromQuery(Name = "pageNumber")] int pageNumber = 0,
        [FromQuery(Name = "pageSize")] int pageSize = 20,
        [FromQuery(Name = "sortBy")] string sortBy = "cui")
    {
        // Your implementation here
        return Ok();
    }
}

File: openapi/code_samples/javascript/api_company_bankrupt_activeYears.js
const express = require('express');
const app = express();
const port = 3000;

app.get('/api/company/bankrupt-companies-active-years', (req, res) => {
    const activityYears = parseInt(req.query.activityYears) || 3;
    const caen = req.query.caen || '';
    const cui = req.query.cui || '';
    const county = req.query.county || '';
    const pageNumber = parseInt(req.query.pageNumber) || 0;
    const pageSize = parseInt(req.query.pageSize) || 20;
    const sortBy = req.query.sortBy || 'cui';

    // Your implementation here

    res.status(200).json({});
});

app.listen(port, () => {
    console.log(`Server is listening at http://localhost:${port}`);
});
